<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>603</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="q_typeBien">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>70</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <item>
    <property name="text">
     <string>Maison</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Appartement</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Château</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Ferme</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Bureau</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Commerce</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="l_typeBien">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Type de bien :</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_superficieTerrain">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>120</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Superficie du terrain :</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="q_superficieTerrain">
   <property name="geometry">
    <rect>
     <x>437</x>
     <y>120</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="maximum">
    <number>100000</number>
   </property>
  </widget>
  <widget class="QLabel" name="l_mCarre">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>120</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>m2</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_nbPieces">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>120</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre de pièces :</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="q_nbPieces">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>120</y>
     <width>51</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="q_adresse">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>160</y>
     <width>381</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="l_adresse">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>160</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Adresse :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="q_codePostal">
   <property name="geometry">
    <rect>
     <x>402</x>
     <y>200</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Code Postal</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_ville">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>200</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Ville :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="q_ville">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>200</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="l_description">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Description du bien :</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="q_description">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>240</y>
     <width>381</width>
     <height>131</height>
    </rect>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
  </widget>
  <widget class="QLabel" name="l_prix">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>380</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Prix :</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="q_prix">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>380</y>
     <width>141</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximum">
    <double>1000000000.990000009536743</double>
   </property>
  </widget>
  <widget class="QLabel" name="l_euros">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>380</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>€</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_photoPrincipale">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>420</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string> Photo principale :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="q_photoPrincipale">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>420</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="b_parcourirPrincipal">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>420</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_typeAnnonce">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Type d'annonce :</string>
   </property>
  </widget>
  <widget class="QComboBox" name="q_typeAnnonce">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>201</width>
     <height>26</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <item>
    <property name="text">
     <string>Location</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Vente</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="b_ajouterPrincipal">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>420</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Ajouter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_ajouterSupp">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>480</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Ajouter</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_photosSupp">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>480</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string> Photos supplémentaires :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_parcourirSupp">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>480</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="q_photosSupp">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>480</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="b_valider">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>550</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Valider</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_annuler">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>550</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Annuler</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_photoPrincipal">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>450</y>
     <width>381</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>510</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
